Source/Private/ASDispatch.mm

```diff
@@ -9,43 +9,27 @@
 #import <AsyncDisplayKit/ASDispatch.h>
 #import <AsyncDisplayKit/ASConfigurationInternal.h>
 
-
 // Prefer C atomics in this file because ObjC blocks can't capture C++ atomics well.
 #import <stdatomic.h>
 
-/**
- * Like dispatch_apply, but you can set the thread count. 0 means 2*active CPUs.
- *
- * Note: The actual number of threads may be lower than threadCount, if libdispatch
- * decides the system can't handle it. In reality this rarely happens.
- */
```

Comment:
These comments are exactly the same as the ones in the header file, so no point in keeping them.